dbt-bigquery/src/dbt/adapters/bigquery/impl.py

```diff
@@ -783,6 +783,10 @@ def get_common_options(
         if labels:
             opts["labels"] = list(labels.items())  # type: ignore[assignment]
 
+        if config.get("resource_tags"):
```

Comment:
nit: you can simplify with a `:=`
```suggestion
        if resource_tags := config.get("resource_tags"):
```

Reply:
Fixed [c0539a4](https://github.com/dbt-labs/dbt-adapters/pull/1177/commits/c0539a4594c7d26c5e71b202c4da787fdf91ffda)